src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index_zh.properties

```diff
@@ -0,0 +1,2 @@
+title=页面标题
+description=应用栏提供了页面标题以及当前页面的重要动作。它必须包含在 <code>l:main-panel</code> 之前。
```

Comment:
did you check this is encoded properly btw? .properties files aren't encoded in utf-8

Reply:
According to `file`, this is a `UTF-8 Unicode text, with CRLF line terminators`

Reply:
Yes but it’s not read as utf-8

I think this is the API but I may be wrong:

https://stackoverflow.com/a/4660195

Reply:
Oic, I'll see if I can take a closer look later with the Locale plugin forcing simplified Chinese.

Reply:
Yeah very broken:

<img width="944" alt="image" src="https://user-images.githubusercontent.com/21194782/166218044-b362742b-cd7f-42f6-bd91-e4472eb64881.png">

Reply:
Oic, that isn't simplified Chinese but some gibberish. I'll see what we can do about that, good finding though 👍🏻

Reply:
This change was not correct, so I think it should be reverted.

Reply:
https://github.com/jenkinsci/stapler/blob/2905144023e5728743ffc2357f506bbc0039f9ee/jelly/src/main/java/org/kohsuke/stapler/jelly/ResourceBundle.java#L155-L165= could be converted to use `PropertyResourceBundle`, which (as of Java 9+, which we will begin requiring in September) [first loads property files in UTF-8, falling back to ISO-8859-1 otherwise](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/PropertyResourceBundle.html).

Reply:
filed https://github.com/jenkinsci/stapler/issues/356

Reply:
FTR, Crowdin implemented a custom solution for us in the meantime, to encode .properties files during the export. The output can be viewed in https://github.com/jenkinsci/design-library-plugin/commit/daca5d5c15b6b7340305bb888d54299ac07fdbff which renders as 
![](https://i.imgur.com/r7kZDa4.png)